src/modules/server.ts

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable no-console */
+import Koa from 'koa';
+import Router from 'koa-router';
+import getPort from 'get-port';
+import asyncRetry from 'async-retry';
+// eslint-disable-next-line no-unused-vars
```

Comment:
Why did you have to use this commented line?

Reply:
For some reason my linting configurations don't count using a variable as only the type of a variable, in this case `server`, so I added this line so that the lint works properly.

Reply:
Fix it, please

Reply:
It was a false positve. I fixed by adding `@vue/typescript/recommended` to extends in the `.eslintrc.json` file